mimesis/enums.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+from random import choice
+
+
+class PortRange(Enum):
+    DEFAULT = (1, 65535)
+    WELL_KNOWN = (1, 1023)
+    EPHEMERAL = (49152, 65535)
+    REGISTERED = (1024, 49151)
+
+
+class Gender(Enum):
+    FEMALE = 'female'
+    MALE = 'male'
+    F = FEMALE
+    M = MALE
+    RANDOM = choice([FEMALE, MALE])
```

Comment:
This is broken, since it selected the gender only once when created.

```
>>> class Gender(Enum):
...    MALE = 'male'
...    FEMALE = 'female'
...    RANDOM = choice([MALE, FEMALE])
... 
>>> Gender.RANDOM
<Gender.MALE: 'male'>
>>> Gender.RANDOM
<Gender.MALE: 'male'>
>>> Gender.RANDOM
<Gender.MALE: 'male'>
>>> Gender.RANDOM
<Gender.MALE: 'male'>
>>> Gender.RANDOM
<Gender.MALE: 'male'>
```